frontend/app/routes/_gcweb-app.data-unavailable.tsx

```diff
@@ -29,8 +29,31 @@ export default function DataUnavailable() {
   const { SCCH_BASE_URI } = getClientEnv();
```

Comment:
Since this .tsx file is a route, you can (and should) use a loader to send config data down to it. Call `getEnv()` in the loader and return `scchBaseUri` from there.

Reply:
updated